gftest.sh

```diff
@@ -0,0 +1,105 @@
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Shouldn't this be the Eclipse Foundation and shouldn't the license be the EPL?

Reply:
Yes, it probably should. I'll update it, thx!